### ISSUE TITLE
DeprecationWarning: `FieldValidationInfo` is deprecated

### ISSUE THREAD

**Issue by arjenzorgdoc** [duo-labs/py_webauthn#191]
When running with `-W err` webauthn raises a `DeprecationWarning`:

```
~/test $ python3 -m venv .venv
~/test $ . ./.venv/bin/activate
(.venv) ~/test $ pip install webauthn
...snip...
Successfully installed annotated-types-0.6.0 asn1crypto-1.5.1 cbor2-5.5.1 cffi-1.16.0 cryptography-41.0.7 pyOpenSSL-23.3.0 pycparser-2.21 pydantic-2.5.2 pydantic-core-2.14.5 typing-extensions-4.8.0 webauthn-1.11.1
(.venv) ~/test $ python -W err -c 'import webauthn'
Traceback (most recent call last):
...snip...
  File ".../.venv/lib/python3.11/site-packages/webauthn/helpers/structs.py", line 6, in <module>
    from pydantic import (  # type: ignore[attr-defined]
...snip...
DeprecationWarning: `FieldValidationInfo` is deprecated, use `ValidationInfo` instead.
(.venv) ~/test $ 
```

**Comment by MasterKale** [comment]
This will end up getting closed out if/when I move forward with #195.

**Comment by MasterKale** [comment]
I've merged #195 which removed Pydantic as a dependency of this project so I'm closing out any PR's and issues related to Pydantic as they're no longer needed.
